expanding-cards-project/js/main.js

```diff
@@ -0,0 +1,8 @@
+const cards = document.querySelectorAll('.card');
+
+cards.forEach(card => {
+    card.addEventListener('click', () => {
+        cards.forEach(c => c.classList.remove('expanded'));
```

Comment:
don't minify variables name
```suggestion
        cards.forEach(card => card.classList.remove('expanded'));
```